docs/CONTRIBUTE.md

```diff
@@ -199,3 +194,8 @@ For contributions we are using [Gitflow as branching strategy](https://www.atlas
 ### Postman
 
 During the development process, it may be helpful to mock Github events. This is possible through the software program [Postman](https://www.postman.com/), which allows developers to create mock HTTP requests sent to the endpoint of their choice. Header and body parameters (where applicable) are fully customizable by the developer. In the `/postman` folder, telemetry-functions contains a file that can be imported into Postman to easily set up the mock HTTP requests.
+
+To use the requests two [global variables](https://learning.postman.com/docs/sending-requests/variables/variables/#defining-global-variables) need to be set.
+
+The first variable is `telemetry_function_url` which for local testing should be set to `http://127.0.0.1:9999/.netlify/functions/metrics`.
```

Comment:
Since you changed the phrasing in the main Readme to reference "the returned URL", is the `:9999` port set, or does one need to adjust the variables for a random port?

Reply:
The port is set automatically. Did the rephrasing because I first planned to also use AWS for local testing too but that is too complicated for now. The solutions I found need docker and I'd like to avoid having docker as an additional dependency but I already modified the documentation.
Now I think it is fine to leave it like this as it is more future proof in case the local technology changes.